modules/apps/foundation/portal/portal-upload/src/main/java/com/liferay/portal/upload/configuration/UploadServletRequestConfiguration.java

```diff
@@ -29,10 +29,16 @@
 )
 public interface UploadServletRequestConfiguration {
 
-	@Meta.AD(deflt = "104857600", required = false)
+	@Meta.AD(
+		deflt = "104857600", description = "max-size-help",
+		name = "overall-maximum-upload-request-size", required = false
```

Comment:
That's something I get confused all the time. There's no consistency in the format we use for the keys. I see both usages of dots and dashes in portal. Since it's an existing lang key (`overall-maximum-upload-request-size` is the old key we have for the configuration in server admin), I just copied it over.